support-frontend/app/controllers/DigitalSubscriptionController.scala

```diff
@@ -4,12 +4,14 @@ import actions.CustomActionBuilders
 import admin.settings.{AllSettings, AllSettingsProvider, SettingsSurrogateKeySyntax}
 import assets.{AssetsResolver, RefPath, StyleContent}
 import com.gu.support.catalog.DigitalPack
-import com.gu.support.config.PayPalConfigProvider
+import com.gu.support.config.{PayPalConfigProvider, StripeConfigProvider}
 import com.gu.support.encoding.CustomCodecs._
 import services.pricing.{PriceSummaryServiceProvider, ProductPrices}
+import com.gu.i18n.Currency.{AUD}
 import com.gu.support.promotions._
 import com.gu.support.zuora.api.ReaderType.{Direct, Gift}
-import config.StringsConfig
+import config.{StringsConfig, RecaptchaConfigProvider}
```

Comment:
Not sure we need to import `StringsConfig` anymore as it looks like you removed `stringsConfig` from the `DigitalSubscriptionController` parameters.